src/scoreboard.js

```diff
@@ -22,6 +22,7 @@
 const ScoreKeeperMod = require('./scorekeeper');
 
 
+
```

Comment:
While you're fixing the other thing, could you remove this extra blank line? :P

Reply:
24 and 25 or just 25?